include/openssl/rsa.h

```diff
@@ -294,10 +295,20 @@ OPENSSL_EXPORT int RSA_meth_set_pub_enc(RSA_METHOD *meth,
 // RSA_meth_set0_app_data sets |app_data| on |meth|. Although set0 functions
 // generally take ownership in AWS-LC, to maintain OpenSSL compatibility,
 // this function does not. It is the consumers responsibility to free
-// |app_data| before |meth| is freed.
+// |app_data|.
 OPENSSL_EXPORT int RSA_meth_set0_app_data(RSA_METHOD *meth, void *app_data);
 
 
+// RSA_meth_set_sign sets |sign| on |meth|. The function |sign| should return
```

Comment:
Do we need more documentation for |sign|? Sould we say that type is a hash_nid and it should create a PKCS1 prefix as in the usage justsmth pointed to https://github.com/aws/aws-lc/blame/612c200ca1ff0f4068fcd47d4869035d16e08bb0/crypto/fipsmodule/rsa/rsa.c#L818? I didn't dive into the implementation to provide an accurate documentation, tbh.